app/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Does this mean use the default prettier config?

Reply:
Yep